esd_services_api_client/beast/v3/_connector.py

```diff
@@ -280,3 +280,17 @@ def get_configuration(
             response.raise_for_status()
 
         return SparkSubmissionConfiguration.from_dict(response.json())
+
+    def get_logs(self, request_id: str) -> Optional[str]:
+        """
+          Returns a deployed SparkJob configuration.
```

Comment:
Returns job configuration or logs? Description does not match the function name.